src/MarketSigner.js

```diff
@@ -253,10 +253,19 @@ function digestPayNow({ auctionId, amount }) {
   });
 }
 
-function digestCashout({ paymentId, iban }) {
```

Comment:
This deprecates a mutation that may be in use, consider supporting the old one until confirmation arrives from clients potentially using it.